slowfs.py

```diff
@@ -64,7 +69,9 @@ def set(self, name, value):
     def _load(self):
         d = {}
         with open(self._path) as f:
-            exec f in d, d
+            # exec f in d, d   # Not Python3 compatible syntax
+            # f is now a io.TextIOWrapper object, whereas "exec" needs a string, bytes or code object => add .read() method
+            six.exec_(f.read(),d)   # Execute code in the scope of globals and locals. code can be a string or a code object. 
```

Comment:
Using six.exec_ is fine and should work on both python 2 and 3.

However using open() is not correct, we should use:

    io.open(self._path, "rb")

Which behave exactly the same in python 2 and 3, opening the file in binary mode.

Again the commented code should be removed, and the comment is not needed and should not be aligned in this way. If you think this needs some documentation, please add a docstring to the function.